### PR TITLE
Add github workflow to validate json

### PR DESCRIPTION
**Pull request by indradhanush**
A couple of times we've accidentally committed invalid JSON which breaks the CLA checks. A GitHub action should help.

### CODE REVIEW

**Review by bobheadxi — APPROVED** [review]
approving to unblock 🆗

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+      # Runs a single command using the runners shell
+      - name: Validate if .clabot is valid JSON
+        run: cat .clabot | jq
+
+      # Runs a set of commands using the runners shell
+      - name: Run a multi-line script
+        run: |
+          echo Add other actions to build,
+          echo test, and deploy your project.
```
Comment (jhchabran) [inline .github/workflows/main.yml:36]:
```suggestion

```

@indradhanush I think this is a leftover from the docs that you want to drop!

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: Validate if .clabot is valid JSON
+        run: cat .clabot | jq
+
```
Comment (indradhanush) [inline .github/workflows/main.yml:31]:
```suggestion
```

**Comment by indradhanush** [comment]
For context, we tested the check in this PR: https://github.com/sourcegraph/clabot-config/pull/50

Link to failed build: https://github.com/sourcegraph/clabot-config/runs/5124232911?check_suite_focus=true

**Pull request merged**